### ISSUE TITLE
[FEATURE] Sample Feature

### ISSUE THREAD

**Issue by eni9889** [eni9889/IG-PP-Issues#2]
Some description of the feature should go here along with any images

**Comment by soh20i** [comment]
Hi,

Can we please have the feature to send a post to all chats at once? (Like an extra button)

**Comment by JMccormick264** [comment]
@soh20i by all chats, do you mean as in all DM chats?

**Comment by soh20i** [comment]
@JMccormick264 Yes. Some people have a lot of chats in Instagram and it takes a lot of time to send a pic/video to them one by one. 
Would be cool to be able to send to all chats at once

**Comment by JMccormick264** [comment]
Please make a [Feature] post

**Comment by soh20i** [comment]
Done :)
